core/src/test/java/io/confluent/kafka/schemaregistry/ClusterTestHarness.java

```diff
@@ -87,7 +86,7 @@ public static int choosePort() {
   protected String zkConnect;
   protected ZkClient zkClient;
   protected ZkUtils zkUtils;
-  protected int zkConnectionTimeout = 6000;
+  protected int zkConnectionTimeout = 30000;
```

Comment:
Will do. Short answer: SASL connections take longer to establish than 6 seconds, sometimes.